src/Elastic.Markdown/Myst/Directives/ImageBlock.cs

```diff
@@ -64,9 +70,10 @@ public class ImageBlock(DirectiveBlockParser parser, ParserContext context)
 	public override void FinalizeAndValidate(ParserContext context)
 	{
 		Label = Prop("label", "name");
-		Alt = Prop("alt");
-		Align = Prop("align");
+		Alt = (Prop("alt") ?? "{undefined}").ReplaceSubstitutions(context);
```

Comment:
Could you please explain what this is doing?

Will this leave a `{undefined}` in the alt property?

Generally, there is valid use case for an empty `alt=""` property. See https://www.w3.org/WAI/tutorials/images/decorative/

Not sure if this still given with this condition.

Reply:
I ~~stole~~ borrowed from [`TabSetBlock.cs`](https://github.com/elastic/docs-builder/blob/main/src/Elastic.Markdown/Myst/Directives/TabSetBlock.cs#L52) 🙃  but if an empty string is better, let's do that!

Reply:
Ah interesting. I also don't understand what it's doing in TabSet :D

Reply:
Uhh I can't think of any. Must have been a leftover from frenzy coding 😸

Reply:
I'm having some trouble figuring out the right syntax. Just changing `"{undefined}"` to `""` is the only thing I could get to work, but I don't think it makes sense to replace subs in an empty string.... Any hints? 😏

Reply:
Maybe something like:

```
Prop("alt")?.ReplaceSubstitutions(context) ?? "";
```

Reply:
Ahhh the `?` after `Prop("alt")` was the missing piece. Thanks!